app/policies/todo_list_policy.rb

```diff
@@ -0,0 +1,49 @@
+class TodoListPolicy < ApplicationPolicy
+  attr_reader :user, :todo_list
+
+  def initialize(user, todo_list)
```

Comment:
check have you added similar code some where? is this logic required here?

Reply:
This code is required because:
It serves as the base setup for all policies.Without it, the policy won't know which user and record it should operate